DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -256,7 +256,7 @@ struct NetworkProtectionStatusView: View {
         .listRowBackground(Color(designSystemColor: .surface))
 
         Section {
-            if #available(iOS 17.0, *) {
+            if #available(iOS 17.8, *) {
```

Comment:
Should this use iOS 18 instead?

```suggestion
            if #available(iOS 18.0, *) {
```

Reply:
This is right, good eye.

I noticed had suggested a change after pushing the commit myself too... more coffeeeeee!